.github/workflows/tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     - name: Run test suite
       run: |
         ./test/wait-for-hawkbit-online
-        ASAN_OPTIONS=fast_unwind_on_malloc=0 dbus-run-session -- pytest -v
+        ASAN_OPTIONS=fast_unwind_on_malloc=0 dbus-run-session -- python -m pytest -v
```

Comment:
How relates this to venv usage?

Reply:
If anyone has an idea *why* that is, I'm also happy to add that.

Reply:
> With `virtualenv` a `venv/bin/pytest` exists, but not with `venv`. I should have probably added that to the commit message.

This is wrong. It has nothing to do with `virtualenv` vs. `venv`. Old setuptools versions seem to ignore the `console_scripts` entry point and do not generate `venv/bin/pytest`. I'll drop this change.

Reply:
Done.